cmake/unit_test_functions.cmake

```diff
@@ -0,0 +1,84 @@
+# A Buccheri 2021. alexanderbuccheri@googlemail.com 
```

Comment:
I typically don't add authorship due to git history but if someone sees this function and copies it (caus it's great) then a ref would be nice